addons/kommander/1.1.x/kommander.yaml

```diff
@@ -0,0 +1,86 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: ClusterAddon
+metadata:
+  name: kommander
+  labels:
+    kubeaddons.mesosphere.io/name: kommander
+  annotations:
+    catalog.kubeaddons.mesosphere.io/addon-revision: "1.0.0-20"
+    appversion.kubeaddons.mesosphere.io/kommander: "1.0.0"
```

Comment:
```suggestion
    appversion.kubeaddons.mesosphere.io/kommander: "1.1.0"
```

Reply:
This would seem to be an appropriate patch.